javascript/widgets/config.js

```diff
@@ -290,6 +290,23 @@ firebaseui.auth.widget.Config.prototype.getTosUrl = function() {
 };
 
 
+/**
+ * @return {boolean} Whether the display name should be displayed.
+ * Defaults to true.
+ */
+firebaseui.auth.widget.Config.prototype.getRequireDisplayName = function() {
+  var signInOptions = this.getSignInOptions_();
+
+  for (var i = 0; i < signInOptions.length; i++) {
+    if (signInOptions[i]['provider'] === firebaseui.auth.idp.EMAIL_PROVIDER_ID
```

Comment:
Also for consistency, we use double equal for string comparisons in the rest of repo.